packages/design-system/src/components/HelpDrawer/HelpDrawer.stories.jsx

```diff
@@ -0,0 +1,92 @@
+import React from 'react';
+import { Title, Subtitle, Description, ArgsTable, PRIMARY_STORY } from '@storybook/addon-docs';
+import { useArgs } from '@storybook/client-api';
+
+import { HelpDrawer as Help } from './HelpDrawer';
```

Comment:
Aliased HelpDrawer because I didn't wanted a single nested story for this component called "Default". 

Went with "Help" because I worried aliasing it to Drawer would be confusing for folks maintaining - since there's another component called Drawer in the system.